format.go

```diff
@@ -409,10 +409,13 @@ func RequestParserAndFormatterFor(format Format, descSource DescriptorSource, em
 // safe for use with InvokeRPC as long as NumResponses and Status are not read
 // until the call to InvokeRPC completes.
 type DefaultEventHandler struct {
-	out        io.Writer
-	descSource DescriptorSource
-	formatter  func(proto.Message) (string, error)
-	verbose    bool
+	Out        io.Writer
+	DescSource DescriptorSource
```

Comment:
I mentioned this in an earlier comment, but this field can simply be removed, instead of widening the exported API of this package with an unused field.

The one place it it used can be change to pass `nil` since the function being invoked (`GetDescriptorText`) no longer needs a descriptor source.

Reply:
Got it, sorry for catching up so slowly on your comments :)